py/picca/delta_extraction/expected_fluxes/dr16_expected_flux.py

```diff
@@ -1103,6 +1159,14 @@ def chi2(eta, var_lss, fudge):
                                       fudge[w],
                                       fill_value="extrapolate",
                                       kind="nearest")
+            self.get_num_pixels = interp1d(self.lambda_[w],
+                                      num_pixels[w],
+                                      fill_value="extrapolate",
+                                      kind="nearest")
+            self.get_valid_fit = interp1d(self.lambda_[w],
```

Comment:
You are interpolating an array of booleans here... Is this what you wanted to do? Where is this function get_valid_fit used?